pkg/sbproxy/reconcile_brokers.go

```diff
@@ -85,6 +88,7 @@ func (r ReconcileBrokersTask) reconcileBrokers(existingBrokers []serviceBrokerRe
 		} else {
 			r.fetchBrokerCatalog(&existingBroker.ServiceBroker)
 		}
+		r.enableServiceAccessVisibilities(&payloadBroker.ServiceBroker)
```

Comment:
Without eventing, this works but I think this is a concern of a "ServiceAccessTask" where it should listen for changes in service access. If you want to register a broker and make all of its services accessible by everyone it's easier this way, but generally it's a separate concern.

Reply:
It makes sense of first thought, but I am not entirely conviced they that register/deresiter should be totally async from enable/disable (when talking CF scenarios for example) , esepcially when we use long-polling.

If we have two cron jobs running (regJob and accessJob) and imagine accessJob runs 1 sec before regJob - we would be trying to enable access for brokers that are still not registered but will be registered in 1 sec) - so accessJob will fail , add a bunch of error logs, and will have to wait one full cycle (basically until the next time the job runs) before actually reconciling the service access. Or access job will have to check if broker is already registered which again is unnecessary overhead and calls)